server/views/resources/how-nonprofit-projects-work.jade

```diff
@@ -0,0 +1,47 @@
+            <h3>Nonprofit Stakeholders</h3>
+            <p>Each nonprofit project was submitted by a nonprofit. A representative from this nonprofit has agreed to serve as a “stakeholder” - an authorative person who understands the organization and its needs for this particular project.</p>
+            <p>Stakeholders have a deep understanding of their organizations’ needs. Campers will work with them to figure out the best solutions to these needs.</p>
+            <p>When you and your pair partner first speak with your nonprofit stakeholder, you’ll:</p>
+            <p>talk at length to better understand their needs. create a new Trello board and use it to prioritize what needs to be built. and establish deadlines based on your weekly time commitment, and how long you think each task will take. Ideally, we’ll scope each project to be completed in 10 sprints. It’s notoriously difficult to estimate how long building software projects will take, so feel free to ask our volunteer team for help.</p>
+            <p>You’ll continue to meet with your stakeholder weekly using the conference software GoToMeeting. You will also correspond with the team on the project’s Trello board.</p>
+            <p>Getting “blocked” on a task can take away your sense of forward momentum, so be sure to proactively seek answers to any ambiguities you encounter.</p>
+            <p>Ultimately, the project will be considered complete once both the stakeholder’s needs have been met, and you and your pair are happy with the project. Then you can add it to your portfolio!</p>
+            <h3>Working with your Pair</h3>
+            <p>You and your pair will pair program (code together on the same computer virtually) about half of the time, and work independently the other half of the time.</p>
+            <p>Here are our recommended ways of collaborating:</p>
+            <p>Gitter has robust private messaging functionality. It’s the main way our team communicates, and we recommend it over email. Trello is great for managing projects. Work with your stakeholder to create Trello cards, and update these cards regularly as you make progress on them. Screen Hero or Team Viewer - These are the ideal way to pair program. Tools like TMUX are good, but difficult to use. We discourage you from using screen sharing tools where only one person has control of the keyboard and mouse - that isn’t real pair programming. Write clear and readable code, commit messages, branch names, and pull request messages.</p>
+            <h3>Hosting Apps</h3>
+            <p>Unless your stakeholder has an existing modern host (AWS, Digital Ocean), you’ll need to transition them over to a new platform. We believe Heroku is the best choice for a vast majority of web projects. It’s free, easy to use, and has both browser and command line interfaces. It’s owned by Salesforce and used by a ton of companies, so it’s accountable and unlikely to go away.</p>
```

Comment:
I take it you mean free to develop on.  It shouldn't be free to host on and there are limitations to how long the app can be active per day on the free plan.